src/basics.php

```diff
@@ -170,9 +130,13 @@ function dd($var, $showHtml = null)
             return;
         }
 
-        debug($var, $showHtml, false);
-        $backtrace = debug_backtrace(false, 1);
-        pr('dd-location: ' . $backtrace[0]['file'] . ':' . $backtrace[0]['line']);
+        $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+        $location = [
+            'line' => $trace[0]['line'],
+            'file' => $trace[0]['file']
+        ];
+
+        Debugger::printVar($var, $location);
```

Comment:
$showHtml got indeed lost in this PR - as mentioned in https://github.com/cakephp/cakephp/pull/10304